experimental/packages/opentelemetry-sdk-metrics-base/src/state/DeltaMetricProcessor.ts

```diff
@@ -35,31 +35,36 @@ export class DeltaMetricProcessor<T extends Maybe<Accumulation>> {
 
   constructor(private _aggregator: Aggregator<T>) {}
 
-  /** Bind an efficient storage handle for a set of attributes. */
-  private bind(attributes: MetricAttributes) {
-    return this._activeCollectionStorage.getOrDefault(attributes, () => this._aggregator.createAccumulation());
-  }
-
-  record(value: number, attributes: MetricAttributes, _context: Context) {
-    const accumulation = this.bind(attributes);
+  record(value: number, attributes: MetricAttributes, _context: Context, collectionTime: HrTime) {
+    const accumulation = this._activeCollectionStorage.getOrDefault(
+      attributes,
+      () => this._aggregator.createAccumulation(collectionTime)
+    );
     accumulation?.record(value);
```

Comment:
I'm curious if the function `getOrDefault` needs to exist at all? Does it have better performance than just using `??` and a fallback like this?

```suggestion
    const accumulation = this._activeCollectionStorage.get(attributes) ?? this._aggregator.createAccumulation(collectionTime);
    accumulation?.record(value);
```

Reply:
As mentioned above, DropAggregator uses `undefined` to represent the accumulated value. We have to check the presence of the attribute key instead of checking the nullish of values here.